airflow/dags/resources/stages/parse/table_definitions/angle/LZagEUR_call_burn.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "inputs": [
+                {
+                    "internalType": "uint256",
+                    "name": "amount",
+                    "type": "uint256"
+                }
+            ],
+            "name": "burn",
+            "outputs": [],
+            "stateMutability": "nonpayable",
+            "type": "function"
+        },
+        "contract_address": "0x0c1ebbb61374da1a8c57cb6681bf27178360d36f",
+        "field_mapping": {},
+        "type": "trace"
+    },
+    "table": {
+        "dataset_name": "angle",
+        "schema": [
+            {
+                "description": "",
+                "name": "amount",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "LZagEUR_call_burn"
```

Comment:
I did not come across the "LZ" naming convention. Since the implementation contract is called `LayerZeroBridgeToken`, maybe `LayerZeroagEUR` would be more descriptive?

Reply:
@TimNooren it is the token symbol 'LZ-agEUR' without the hyphen. Made the change to the more descriptive LayerZeroagEUR

Reply:
@AyadiGithub Oh right thanks, I missed that. Just wanted to make sure we're not abbreviating unnecessarily:)